docsite-css/component-display-conatiner.css

```diff
@@ -24,6 +24,10 @@
   width: 50%;
   margin: auto;
 }
+.component-display-container .form-grp {
+  margin-left: auto;
+  margin-right: auto;
+}
```

Comment:
Improvment: You give margins like { margin: 0 auto }.

Reply:
Ohh that is right. I will do that 😄